.evergreen/config/functions.yml

```diff
@@ -185,28 +195,32 @@ functions:
   "run OCSP responder":
     - command: shell.exec
       params:
+        include_expansions_in_env:
+          - TEST_COLUMN
+          - CERT_TYPE
+          - USE_DELEGATE
         shell: bash
         script: |
           ${PREPARE_SHELL}
-          TEST_COLUMN=${TEST_COLUMN} \
-          CERT_TYPE=${CERT_TYPE} \
-          USE_DELEGATE=${USE_DELEGATE} \
           ${PROJECT_DIRECTORY}/.evergreen/run-ocsp-responder.sh
 
   "run tests":
     - command: shell.exec
       type: test
       params:
+        include_expansions_in_env:
+          - API_VERSION
+          - ATLAS_CONNECTIVITY_URIS
```

Comment:
Should this always be specified? In the `test-atlas-connectivity` task, `run tests` is invoked with `vars: { TESTS: "tests/atlas.phpt" }`, so I would expect that's the only case where you'd actually want to pass this potentially sensitive environment variable onwards.